Sources/App/commands.swift

```diff
@@ -122,11 +122,14 @@ extension SlackCommand {
         // [CNSMR-1319] TODO: Use a config file to parametrise accountable person
         let isTelus = release.appName?.caseInsensitiveCompare("Telus") == .orderedSame
         let accountablePerson = isTelus ? "eric.schnitzer" : "andreea.papillon"
+        // Remove brackets around JIRA ticket names so that it's recognized by JIRA as a ticket reference
+        // eg replace "[CNSMR-123] Do this" with "CNSMR-123 Do this"
+        let cleanChangelog = changelog.replacingOccurrences(of: "\\[([A-Z]+-[0-9]+)\\]", with: "$1", options: [.regularExpression], range: nil)
```

Comment:
this can be just shadowed
```suggestion
        let changelog = changelog.replacingOccurrences(of: "\\[([A-Z]+-[0-9]+)\\]", with: "$1", options: [.regularExpression], range: nil)
```